### PR TITLE
835299-Prepared migration page for SfNumericEntry in .NET MAUI

### PR DESCRIPTION
**Pull request by AarthiArjunan**
Description:
Here I have added the Migration content in the .NET MAUI SfNumericEntry control.

### CODE REVIEW

File: MAUI/NumericEntry/Migration.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+<th>Xamarin SfNumericTextBox</th>
+<th>.NET MAUI SfNumericEntry</th></tr>
+<tr>
+<td>Syncfusion.XForms.SfNumericTextBox</td>
+<td>Syncfusion.Maui.Inputs</td></tr>
+</table>
+
+## Properties
+
+<table>
```
Comment (anandhan-rajagopal) [inline MAUI/NumericEntry/Migration.md:27]: Add all available API links

File: MAUI/NumericEntry/Migration.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfNumericEntry | Syncfusion 
+description: Learn about Migrating from Syncfusion Xamarin NumericTextBox to Syncfusion .NET MAUI NumericEntry control and more.
```
Comment (anandhan-rajagopal) [inline MAUI/NumericEntry/Migration.md:4]: Learn about Migrating from Syncfusion Xamarin NumericTextBox and NumericUpDown to Syncfusion .NET MAUI NumericEntry control and more.

File: MAUI/NumericEntry/Migration.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+platform: maui
+control: SfNumericEntry
+documentation: ug
+---  
+
+# Migrating from Xamarin SfNumericTextBox to .NET MAUI SfNumericEntry
```
Comment (anandhan-rajagopal) [inline MAUI/NumericEntry/Migration.md:10]: Migrating from Xamarin SfNumericTextBox  and SfNumericUpDown to .NET MAUI SfNumericEntry

Comment (anandhan-rajagopal) [reply MAUI/NumericEntry/Migration.md:10]: Add SfNumericUpDown in all places

File: MAUI/NumericEntry/Migration.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+To migrate easier from the Xamarin SfNumericTextBox to `.NET MAUI SfNumericEntry`, most of the APIs are kept from the Xamarin SfNumericTextBox in MAUI SfNumericEntry. However, to maintain the consistency of API naming in MAUI SfNumericEntry, rename some of the APIs. The APIs changed in MAUI SfNumericEntry from Xamarin SfNumericTextBox are detailed as follows.
+
+## Namespaces 
+
+<table>
+<tr>
+<th>Xamarin SfNumericTextBox</th>
+<th>.NET MAUI SfNumericEntry</th></tr>
+<tr>
+<td>Syncfusion.XForms.SfNumericTextBox</td>
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/NumericEntry/Migration.md:21]: Addressed the changes

File: MAUI/NumericEntry/Migration.md (hunk 2 above)
Comment (AarthiArjunan) [reply MAUI/NumericEntry/Migration.md:4]: Addressed the changes.

File: MAUI/NumericEntry/Migration.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+title: Migrating from Xamarin to .NET MAUI SfNumericEntry | Syncfusion 
+description: Learn about Migrating from Syncfusion Xamarin NumericTextBox and NumericUpDown to Syncfusion .NET MAUI NumericEntry control and more.
+platform: maui
+control: SfNumericEntry
+documentation: ug
+---  
+
+# Migrating from Xamarin SfNumericTextBox and SfNumericUpDown to .NET MAUI SfNumericEntry
+
+To migrate easier from the Xamarin SfNumericTextBox and SfNumericUpDown to `.NET MAUI SfNumericEntry`, most of the APIs are kept from the Xamarin SfNumericTextBox and SfNumericUpDown  in MAUI SfNumericEntry. However, to maintain the consistency of API naming in MAUI SfNumericEntry, rename some of the APIs. The APIs changed in MAUI SfNumericEntry from Xamarin SfNumericTextBox and SfNumericUpDown are detailed as follows.
```
Comment (florenceliliana) [inline MAUI/NumericEntry/Migration.md:12]: To migrate easier from the Xamarin SfNumericTextBox and SfNumericUpDown to `.NET MAUI SfNumericEntry`, most of the APIs are kept from the Xamarin SfNumericTextBox and SfNumericUpDown in MAUI SfNumericEntry. However, rename some of the APIs to maintain the consistency of API naming in MAUI SfNumericEntry. The APIs changed in MAUI SfNumericEntry from Xamarin SfNumericTextBox and SfNumericUpDown are detailed as follows.

File: MAUI/NumericEntry/Migration.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+<table> 
+<tr>
+<th>Xamarin SfNumericTextBox <br> Xamarin SfNumericUpDown </th>
+<th>.NET MAUI SfNumericEntry</th>
+<th>Description</th></tr>
+<tr>
+<td>[BorderColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_BorderColor)</td>
+<td>[Stroke](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_Stroke)</td>
+<td>Gets or sets a color that describes the stroke.</td></tr>
```
Comment (florenceliliana) [inline MAUI/NumericEntry/Migration.md:35]: <td>Get or set a color that describes the stroke.</td></tr>

File: MAUI/NumericEntry/Migration.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+<tr>
+<td>[AllowDefaultDecimalDigits](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_AllowDefaultDecimalDigits)</td>
+<td>[AllowNull](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_AllowNull)</td>
+<td>Gets or sets a value indicating whether the SfNumericEntry allows null value or not.</td></tr>
```
Comment (florenceliliana) [inline MAUI/NumericEntry/Migration.md:39]: <td>Get or set a value indicating whether the SfNumericEntry allows a null value or not.</td></tr>

File: MAUI/NumericEntry/Migration.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+<tr>
+<td>[ClearButtonVisibility](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_ClearButtonVisibility)</td>
+<td>[ShowClearButton](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_ShowClearButton)</td>
+<td>Gets or sets a value indicating whether to show or hide the clear button in SfNumericEntry.</td></tr>
```
Comment (florenceliliana) [inline MAUI/NumericEntry/Migration.md:43]: <td>Get or set a value indicating whether to show or hide the clear button in the SfNumericEntry.</td></tr>

File: MAUI/NumericEntry/Migration.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+<tr>
+<td>[FormatString](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_FormatString)</td>
+<td>[CustomFormat](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_CustomFormat)</td>
+<td>Gets or sets the format used to specify the formatting of Value.</td></tr>
```
Comment (florenceliliana) [inline MAUI/NumericEntry/Migration.md:47]: <td>Get or set the format used to specify the formatting of Value.</td></tr>

File: MAUI/NumericEntry/Migration.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+<tr>
+<td>[IsReadOnly](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_IsReadOnly)</td>
+<td>[IsEditable](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_IsEditable)</td>
+<td>Gets or sets a value indicating whether the SfNumericEntry allows editing the value or not.</td></tr>
```
Comment (florenceliliana) [inline MAUI/NumericEntry/Migration.md:51]: <td>Get or set a value indicating whether the SfNumericEntry allows editing the value or not.</td></tr>

File: MAUI/NumericEntry/Migration.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+<tr>
+<td>[Watermark](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_Watermark)</td>
+<td>[Placeholder](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_Placeholder)</td>
+<td>Gets or sets the text that is displayed in the control until the value is changed by a user action or some other operation.</td></tr>
```
Comment (florenceliliana) [inline MAUI/NumericEntry/Migration.md:55]: <td>Get or set the text that is displayed in the control until the value is changed by a user action or some other operation.</td></tr>

File: MAUI/NumericEntry/Migration.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+<tr>
+<td>[WatermarkColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_WatermarkColor)</td>
+<td>[PlaceholderColor](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_PlaceholderColor)</td>
+<td>Gets or sets the text that is displayed in the control until the value is changed by a user action or some other operation.</td></tr>
```
Comment (florenceliliana) [inline MAUI/NumericEntry/Migration.md:59]: <td>Get or set the text that is displayed in the control until the value is changed by a user action or some other operation.</td></tr>

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The review is completed..

File: MAUI/NumericEntry/Migration.md (hunk 5 above)
Comment (anandhan-rajagopal) [reply MAUI/NumericEntry/Migration.md:12]: changes are done

File: MAUI/NumericEntry/Migration.md (hunk 7 above)
Comment (anandhan-rajagopal) [reply MAUI/NumericEntry/Migration.md:39]: changes are done

File: MAUI/NumericEntry/Migration.md (hunk 9 above)
Comment (anandhan-rajagopal) [reply MAUI/NumericEntry/Migration.md:47]: changes are done

File: MAUI/NumericEntry/Migration.md (hunk 10 above)
Comment (anandhan-rajagopal) [reply MAUI/NumericEntry/Migration.md:51]: changes are done

File: MAUI/NumericEntry/Migration.md (hunk 11 above)
Comment (anandhan-rajagopal) [reply MAUI/NumericEntry/Migration.md:55]: changes are done

File: MAUI/NumericEntry/Migration.md (hunk 12 above)
Comment (anandhan-rajagopal) [reply MAUI/NumericEntry/Migration.md:59]: changes are done

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**